### PR TITLE
OPENCAST-2429 - fallback recorder ingestion

### PR DESCRIPTION
**Pull request by NgoniChoga**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -43,8 +43,8 @@ <h2>Start ad-hoc recording</h2>
   <main>
     <header>
       <a href="https://media.uct.ac.za/">Admin Login</a
-      ><a href="http://camonitor.uct.ac.za:3000/" target="_blank">IP Cameras</a
-      ><a href="http://camonitor.uct.ac.za/ls/ls-status.txt" target="_blank">Lecturesight</a
+      ><a href="http://camonitor.uct.ac.za/ip-cam/" target="_blank">IP Cameras</a
```
Comment (smarquard) [inline public/index.html]: These are all https URLs now

**Comment by smarquard** [comment]
Does package-lock.json need to be in the repo?

File: public/index.html (hunk 1 above)
Comment (NgoniChoga) [reply public/index.html]: URLs have been updated.

**Comment by NgoniChoga** [comment]
package-lock.json is intended to be committed into source repositories.
Ref: https://docs.npmjs.com/files/package-lock.json

**Review by smarquard — APPROVED** [review]

**Comment by smarquard** [comment]
For some reason these 4 GSH venues are hard-coded in the index.html so they show up first in the list. They should be removed (no reason to treat them specially).

        <select>
          <option value="">--Choose capture agent--</option>
          <option value="clinskills1">GSH Clinical Skills Lab 1</option>
          <option value="clinskills2">GSH Clinical Skills Lab 2</option>
          <option value="clinskills3">GSH Clinical Skills Lab 3</option>
          <option value="clinskills4">GSH Clinical Skills Lab 4</option>
        </select>

**Pull request merged**